portfolio/src/test/java/com/google/sps/NameTest.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.servlets;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+@RunWith(JUnit4.class)
+public class NameTest {
+
+  private final DataServlet servlet = new DataServlet();
+
+  @Rule public MockitoRule mockito = MockitoJUnit.rule();
+
+  @Mock private HttpServletRequest request;
+  @Mock private HttpServletResponse response;
+
+  @Test
+  public void getMyName() throws IOException {
+    StringWriter stringWriter = new StringWriter();
+    PrintWriter printWriter = new PrintWriter(stringWriter);
+    when(response.getWriter()).thenReturn(printWriter);
+    servlet.doGet(request, response);
+
+    verify(response).setContentType("text/plain;");
+    assertTrue(stringWriter.toString().contains("Hello Bianca!"));
```

Comment:
nit - consider switching to the [Google Truth library](https://github.com/google/truth) - it has more fluent assertions (and i think will be easier to use in the capstone project).

Reply:
also, in general, `equals` (or `isEqualTo` in truth) would be a stronger assertion here than `contains`.